destination_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2022 Meroxa, Inc.
+// Copyright © 2023 Meroxa, Inc.
```

Comment:
I'm not sure if I remember right, but didn't we agree on leaving the original copyright year when updating a file? not a big deal ofc just asking

Reply:
It was done automatically by the IDE, I think it thought it's a new file and updated the license. I'll revert it 👍